src/prog_models/models/uav_model/uav_model.py

```diff
@@ -0,0 +1,556 @@
+        | Model-defined inputs: 
+            | T: thrust
+            | mx: moment in x 
+            | my: moment in y
+            | mz: moment in z
+
+    :term:`States<state>`: (13)
+        | x: cartesian position
+        | y: cartesian position
+        | z: cartesian position
+        | phi: pitch
+        | theta: roll
+        | psi: yaw
+        | vx: velocity 
+        | vy: velocity 
+        | vz: velocity 
+        | p: angular velocity 
+        | q: angular velocity 
+        | r: angular velocity 
+        | t: time 
+
+    :term:`Outputs<output>`: (12)
+        | x: cartesian position
+        | y: cartesian position
+        | z: cartesian position
+        | phi: pitch
+        | theta: roll
+        | psi: yaw
+        | vx: velocity 
+        | vy: velocity 
+        | vz: velocity 
+        | p: angular velocity 
+        | q: angular velocity 
+        | r: angular velocity 
+        | t: time 
+
+    Keyword Args
+    ------------
+        process_noise : Optional, float or dict[str, float]
+          :term:`Process noise<process noise>` (applied at dx/next_state). 
+          Can be number (e.g., .2) applied to every state, a dictionary of values for each 
+          state (e.g., {'x1': 0.2, 'x2': 0.3}), or a function (x) -> x
+        process_noise_dist : Optional, str
+          distribution for :term:`process noise` (e.g., normal, uniform, triangular)
+        measurement_noise : Optional, float or dict[str, float]
+          :term:`Measurement noise<measurement noise>` (applied in output eqn).
+          Can be number (e.g., .2) applied to every output, a dictionary of values for each
+          output (e.g., {'z1': 0.2, 'z2': 0.3}), or a function (z) -> z
+        measurement_noise_dist : Optional, str
+          distribution for :term:`measurement noise` (e.g., normal, uniform, triangular)
+        flight_file : Optional, str
+          Text file to specify waypoint information. Necessary columns must be in the following
+          order with units specified: latitude (labeled 'lat_deg' or 'lat_rad'), longitude 
+          (labeled 'lon_deg' or 'lon_rad'), and altitude (labeled 'alt_ft' or 'alt_m'). An 
+          additional column for ETAs may be included (labeled 'time_unix). Note that while
+          'flight_file' is optional, either 'flight_file' or 'flight_plan' must be specified.
+        flight_plan : Optional, dict[str, numpy array]
+          Dictionary to specify waypoint information. Necessary keys must include the following
+          with units specified: latitude ('lat_deg' or 'lat_rad'), longitude 
+          (labeled 'lon_deg' or 'lon_rad'), and altitude (labeled 'alt_ft' or 'alt_m'). An 
+          additional key for ETAs may be included (labeled 'time_unix). Each key must correspond
+          to a numpy array of values. Note that while 'flight_plan' is optional, either 
+          'flight_file' or 'flight_plan' must be specified.
+        flight_name : Optional, str
```

Comment:
What do we mean by name to identify flight plan? I see that it's passed into trajectory and route building, but what does it add?

Reply:
same with aircraft_name

Reply:
Good point. It doesn't add anything in terms of the actual trajectory generation. The original code had capability to save results to a file, and I believe the name(s) were used for specifying the file name. I didn't bring that capability over, so it's not used here. @portia19 or @matteocorbetta, can you confirm I'm understanding this correctly? Are we okay with not including the saving capabilities (users could use our ProgPy serialization techniques instead) and getting rid of aircraft_name and flight_name?

Reply:
I've removed flight_name and aircraft_name in the new PR with modularity changes, so this will be adjusted soon